### DISCUSSION TITLE
How can I access all tenant data at once?

### DISCUSSION THREAD

**Discussion by monish-khatri** [spatie/laravel-multitenancy#410] [Q&A]
Suppose I have `tenant_1` and `tenant_2`

## Tenant 1
- Users
  - Test User 1
  - Test User 2
## Tenant 2
- Users
  - Test User 3
  - Test User 4

What I Want is to show all the users at once for admin view

Output : 
- Users
  - Test User 1
  - Test User 2
  - Test User 3
  - Test User 4

Is any feasible solution to achieve the above scenario?
Any help will be Usefull.

**Comment by masterix21** [comment]
What about a database view?

**Reply by monish-khatri** [reply]
> What about a database view?

There will be different database for `Tenant 1` and `Tenant 2` and both database contains `users` table.

**Comment by denizgolbas** [comment]
Create a landlord users_combine table and connect all tenant user databases with fdw.
